src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/ManagementPartnerTests.ps1

```diff
@@ -70,14 +88,22 @@ Tests update management partner
 #>
 function Test-UpdatePartner
 {
-	$partnerId="123456"
-	$partner = Update-AzureRmManagementPartner -PartnerId $partnerId
+	# Setup
+	$partnerId="123457"
+	$partner = New-AzureRmManagementPartner -PartnerId $partnerId
+
+    # Test
+    $newParnterId="123456"
```

Comment:
Typo here: $newPar**nt**erId should be $newPar**tn**erId (and in the 2 uses of it below)

Reply:
Good catch